Add new column to a materialized view revision (#173)
tinybirdco/use-case-examples: add_new_column_to_a_materialized_view/README.md

For that the steps will be:

1. Change the Materialized View (Pipe and Data Source) to add the new column.

> 2. Bump version from 0.0.0 -> 1.0.0. Bumping the major version, it will create a new `Preview Release`` internally forking the Materialized View and its dependencies.

Reviewer: double backticks at the end

> 2. Bump version from 0.0.0 -> 1.0.0. Bumping the major version, it will create a new `Preview Release`` internally forking the Materialized View and its dependencies.

Reviewer: ```suggestion
2. Bump version from 0.0.0 -> 1.0.0. Bumping the major version, it will create a new `Preview Release` internally forking the Materialized View and its dependencies.
```

> 3. Backfill the `Preview Release`` Materialized View with the data previous to its creation.

Reviewer: same here

> 3. Backfill the `Preview Release`` Materialized View with the data previous to its creation.

Reviewer: ```suggestion
3. Backfill the `Preview Release` Materialized View with the data previous to its creation.
```

- Get the creation date by executing the following command
```sh
tb --semver 1.0.0 datasource ls
```
or executing this query using the CLI or the UI dashboard

```sh
tb --semver 1.0.0 sql "select timestamp from tinybird.datasources_ops_log where event_type = 'create' and datasource_name = 'analytics_sessions_mv' order by timestamp desc limit 1"
```

- Use the creation date to populate the Materialized View with the data previous to its creation. You can run the next command directly or inside a custom deployment:
```sh
tb --semver 1.0.0 pipe populate analytics_sessions --node analytics_sessions_1 --sql-condition "timestamp < '$CREATED_AT' --wait
```

## 4: Promote the changes
Once the Materialized View has been already populated you can promote the `Preview Release` to `Live`. Choose one of the next 3 options for that:

- If you are using our workflow templates just run the action `Tinybird - Releaseas Workflow` in other case you can use the command:

- Run the following command from the CLI:
  
```sh
  tb release promote --semver 1.0.0
```

> - Go the `Releases` section on the UI and promote the `Preview 1.0.0``

Reviewer: ```suggestion
- Or go to the `Releases` section in the UI and promote the `Preview 1.0.0`
```
